README.md

```diff
@@ -35,40 +35,39 @@ structure defined in the default stack.
 
 ### Prerequisites
  - Python 3.8+
- - [Cookiecutter Python package](http://cookiecutter.readthedocs.org/en/latest/installation.html) >= 2.1.0: This can be installed with pip:
+ - [Databricks CLI](https://docs.databricks.com/en/dev-tools/cli/databricks-cli.html) >= v0.203.2
 
-``` bash
-$ pip install 'cookiecutter>=2.1.0'
-```
+[Databricks CLI](https://docs.databricks.com/en/dev-tools/cli/databricks-cli.html) v0.203.2 contains [Databricks asset bundle templates](https://docs.databricks.com/en/dev-tools/bundles/templates.html) for the purpose of project creation.
+Databricks asset bundle is in private preview. Please contact a Databricks representative for more information.
 
 ### Starting a new project
 
 To create a new project, run:
 
-    cookiecutter https://github.com/databricks/mlops-stack
+    databricks bundle init https://github.com/databricks/mlops-stack
 
 This will prompt for parameters for project initialization. Some of these parameters are required to get started:
- * ``project_name``: name of the current project
- * ``root_dir__update_if_you_intend_to_use_monorepo``: name of the root directory. The root directory name should be specified if monorepo will be used for the project. 
-   Otherwise, user can leave it blank to use the name of the current project as root directory name.
- * ``cloud``: Cloud provider you use with Databricks (AWS, Azure, or GCP)
- * ``cicd_platform`` : CI/CD platform of choice (GitHub Actions or GitHub Actions for GitHub Enterprise Servers or Azure DevOps)
+ * ``input_project_name``: name of the current project
+ * ``input_root_dir``: name of the root directory. The root directory name should be specified if monorepo will be used for the project. 
+   Otherwise, user can use the name of the current project as root directory name.
```

Comment:
Leaving it blank is no longer an option? We should make it clearer:
```suggestion
 * ``input_root_dir``: name of the root directory. It is recommended to use the name of the current project as the root directory name, except in the case of a monorepo with other projects where the name of the monorepo should be used instead.
```

Reply:
Yes. Leave it black will use the default value `my-mlops-project`. Good suggestion.